docs/topics/sb-features-and-frameworks-summary.adoc

```diff
@@ -57,7 +57,7 @@ a|
 Server Side Load Balancing
 
 a|
-Handle load increases by deploying multiple instances of a service, transparently distributing the load across instances.
+Handles load increases by deploying multiple instances of a service, it transparently distributes the load across instances.
```

Comment:
This reads wrong IMHO. Instead of "it", should there be "and"? Or actually, should it be more like "Handles load increases by deploying multiple service instances and transparently distributing the load across the instances"?

Reply:
@Ladicek sounds good. I'll apply this change. Thanks

Reply:
May I suggest a slight rewording of @Ladicek's version?

_Handles load increases by deploying multiple service instances, and by transparently distributing the load across them._

Reply:
@tradej thanks for your suggestion. Done. LGTM